core/src/main/scala/org/apache/spark/SSLOptions.scala

```diff
@@ -32,9 +49,8 @@ private[spark] case class SSLOptions(
     protocol: Option[String] = None,
     enabledAlgorithms: Set[String] = Set.empty) {
 
-  /**
-   * Creates a Jetty SSL context factory according to the SSL settings represented by this object.
-   */
+  /** Creates a Jetty SSL context factory according to the SSL settings represented by this object.
+    */
```

Comment:
Unfortunately, I can't because Spark tests will fail - they are strictly checking the right margin.